src/math.rs

```diff
@@ -1,11 +1,21 @@
 use vecmath::Matrix4;
 use quaternion::Quaternion;
+use std::mem;
+
+pub fn inv_sqrt(x: f32) -> f32 {
```

Comment:
I mostly added it since the `rsqrt` in `std::num::Float` is deprecated, and wasn't sure what else to use.